src/bio2bel/sources/intact.py

```diff
@@ -0,0 +1,404 @@
+    :return: filtered list of pubmed ids
+    """
+    final_list = []
+    for publications in publication_ids:
+        publications_list = split_to_list(publications, separator='|')
+        flag = False
+        for i in publications_list:
+            if i.startswith('pubmed:'):
+                final_list.append(i)
+                flag = True
+        if not flag:
+            final_list.append('no pubmed id')
+
+    return final_list
+
+
+def filter_for_pubmed(df: pd.DataFrame, column_name: str):
+    """Filter the publication ids for pubmed ids.
+
+    :param df: dataframe
+    :param column_name: column with publication ids
+    :return: dataframe with filtered column
+    """
+    pubmed_list = list_pubmed(df.loc[:, column_name])
+    df = add_to_df(df=df, column_name=column_name, list_to_add=pubmed_list)
+    return df
+
+
+def add_to_df(df: pd.DataFrame, column_name: str, list_to_add: List) -> pd.DataFrame:
+    """Add a column to a dataframe.
+
+    :param df: dataframe
+    :param column_name: column name
+    :param list_to_add: list to be added in column in the dataframe
+    :return: df with updated column
+    """
+    df.loc[:, column_name] = list_to_add
+    return df
+
+
+def get_processed_intact_df() -> pd.DataFrame:
+    """Load, filter and rename intact dataframe.
+
+    :return: processed dataframe
+    """
+    print(_load_file())
+    # original intact dataframe
+    df = _get_my_df()
+
+    # rename columns
+    df = rename_columns(df=df, columns_mapping=columns_mapping)
+
+    # initally preprocess intact file
+    df = read_intact_file(df)
+
+    # filter for pubmed
+    df = filter_for_pubmed(df, PUBLICATION_ID)
+
+    return df
+
+
+# TODO: add edges
+
+
+def get_bel() -> BELGraph:
+    """Get BEL graph.
+
+    :return: BEL graph#
+    """
+    df = _get_my_df()
+    graph = BELGraph(name='intact')
+    for _, row in df.iterrows():
+        _add_my_row(graph, row)
+    return graph
+
+
+def _add_my_row(graph: BELGraph, row) -> None:
+    """Add for every pubmed ID an edge with information about relationship type, source and target.
+
+    :param graph: graph to add edges to
+    :param row: row with metainformation about source, target, relation, pubmed_id
+    :return: None
+    """
+    relation = row[RELATION]
+    source_uniprot_id = row[SOURCE]
+    target_uniprot_id = row[TARGET]
+
+    pubmed_ids = row['pubmed_ids']
+    # pubmed_ids = pubmed_ids.split('|')
+
+    source = pybel.dsl.Protein(
+        namespace='uniprot',
+        identifier=source_uniprot_id,
+        name=get_mnemonic(source_uniprot_id),
+    )
+    target = pybel.dsl.Protein(
+        namespace='uniprot',
+        identifier=target_uniprot_id,
+        name=get_mnemonic(target_uniprot_id),
+    )
+
+    for pubmed_id in pubmed_ids:
+
+        # INCREASE
+        if relation in INTACT_INCREASES_ACTIONS:
+
+            if relation == 'phosphorylation reaction':
+                target_ub = target.with_variants(
```

Comment:
calling all of these `target_ub` might be a bit misleading. maybe target_mod is better?

Reply:
Yes, makes more sense!

Reply:
Fixed: 2e2d5cc57a2d6cefb2c2f4a025f388bb0eab871e